lib/query-utils.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const cleanQuery = (query) => {
+  const regex = /(\()[\sa-zA-Z0-9,\:\"\'\{\}\$\!]+(\))/g
+
+  return query.replace(regex, '$1' + '***' + '$2')
```

Comment:
instead of separating out the matches, could prob grab the whole section and just replace with '(***)' so you don't need to concat.